vignettes/transformers.Rmd

```diff
@@ -33,28 +33,39 @@ aim right now is not to include most of these custom functions within the
 transformers to fit their individual needs.
 
 ```{r, include = FALSE}
-library(glue)
 knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
 ```
 
+```{r}
+library(glue)
+```
+
 ### collapse transformer
 
 A transformer which automatically collapses any glue block ending with `*`.
 
 ```{r}
 collapse_transformer <- function(regex = "[*]$", ...) {
   function(text, envir) {
-    if (grepl(regex, text)) {
-        text <- sub(regex, "", text)
+    collapse <- grepl(regex, text)
+    if (collapse) {
+      text <- sub(regex, "", text)
+    }
+    res <- identity_transformer(text, envir)
```

Comment:
Should the `...` appear here? I've been uncertain about using the `...` in these transformer examples. Re: whether they are for controlling the base case, the "transformer" case, or both.

Reply:
I think it depends on the example, but in this case I think you have it in the right spot.